platform-emf/src/main/java/com/softicar/platform/emf/token/parser/EmfTokenMatrixParser.java

```diff
@@ -34,13 +36,20 @@
 	public EmfTokenMatrixParser(IEmfTable<R, P, ?> table) {
 
 		this.table = Objects.requireNonNull(table);
+		this.fields = table.getAllFields();
 		this.converters = new EmfTokenConverters();
 		this.currentRowIndex = null;
 		this.currentRow = null;
 		this.currentColumnIndex = null;
 		this.currentToken = null;
 	}
 
+	public EmfTokenMatrixParser<R, P> setFields(Collection<IDbField<R, ?>> fields) {
```

Comment:
We should add Javadoc here (all other public parts of this class have some)

Reply:
done